checkstyle.xml

```diff
@@ -17,7 +17,7 @@
 
```

Comment:
Are the changes in this file to handle errors from the checkstyle plugin? If so, for now, we're downgrading the checkstyle plugin to version 6.19 in all repos to get past the error. Can you try that instead of changing the checkstyle xml?

Reply:
Downgraded to 6.19